# Nonexistence of a distance-regular graph with intersection array $\{55, 54, 50, 35, 10; 1, 5, 20, 45, 55\}$

We will show that a distance-regular graph with intersection array $\{55, 54, 50, 35, 10; 1, 5, 20, 45, 55\}$ does not exist. The existence of such a graph would give a counterexample to a conjecture by MacLean and Terwilliger, see [Bipartite distance-regular graphs: The $Q$-polynomial property and pseudo primitive idempotents](http://dx.doi.org/10.1016/j.disc.2014.04.025) by M. Lang.

In [1]:
import drg

Such a graph would be bipartite with $3500$ vertices.

In [2]:
p = drg.DRGParameters([55, 54, 50, 35, 10], [1, 5, 20, 45, 55])
print(p.is_bipartite())
print(p.order())

True
3500


We see that all intersection numbers are nonnegative integers.

In [3]:
p.pTable()

0: [   1    0    0    0    0    0]
   [   0   55    0    0    0    0]
   [   0    0  594    0    0    0]
   [   0    0    0 1485    0    0]
   [   0    0    0    0 1155    0]
   [   0    0    0    0    0  210]

1: [  0   1   0   0   0   0]
   [  1   0  54   0   0   0]
   [  0  54   0 540   0   0]
   [  0   0 540   0 945   0]
   [  0   0   0 945   0 210]
   [  0   0   0   0 210   0]

2: [   0    0    1    0    0    0]
   [   0    5    0   50    0    0]
   [   1    0  243    0  350    0]
   [   0   50    0 1260    0  175]
   [   0    0  350    0  805    0]
   [   0    0    0  175    0   35]

3: [  0   0   0   1   0   0]
   [  0   0  20   0  35   0]
   [  0  20   0 504   0  70]
   [  1   0 504   0 980   0]
   [  0  35   0 980   0 140]
   [  0   0  70   0 140   0]

4: [   0    0    0    0    1    0]
   [   0    0    0   45    0   10]
   [   0    0  180    0  414    0]
   [   0   45    0 1260    0  180]
   [   1    0  414    0  740    0]
   [   0   10    0  180    0   20]

5: [  0   0   0  

The Krein parameters are also nonnegative. Many Krein parameters are zero - in particular, the graph would be $Q$-polynomial for the natural ordering of the eigenvalues.

In [4]:
p.kreinParameters()

0: [   1    0    0    0    0    0]
   [   0  132    0    0    0    0]
   [   0    0 1617    0    0    0]
   [   0    0    0 1617    0    0]
   [   0    0    0    0  132    0]
   [   0    0    0    0    0    1]

1: [     0      1      0      0      0      0]
   [     1   50/3  343/3      0      0      0]
   [     0  343/3 2450/3    686      0      0]
   [     0      0    686 2450/3  343/3      0]
   [     0      0      0  343/3   50/3      1]
   [     0      0      0      0      1      0]

2: [     0      0      1      0      0      0]
   [     0   28/3  200/3     56      0      0]
   [     1  200/3 2380/3    700     56      0]
   [     0     56    700 2380/3  200/3      1]
   [     0      0     56  200/3   28/3      0]
   [     0      0      0      1      0      0]

3: [     0      0      0      1      0      0]
   [     0      0     56  200/3   28/3      0]
   [     0     56    700 2380/3  200/3      1]
   [     1  200/3 2380/3    700     56      0]
   [     0   28/3  200/3     56    

We check the remaining known feasibility conditions. We skip the sporadic nonexistence check since the intersection array is already included.

In [5]:
p.check_feasible(skip = ["sporadic"])

We now compute the triple intersection numbers with respect to three vertices $u, v, w$ at mutual distances $2$. Note that we have $p^2_{22} = 5$, so such triples must exist. The parameter $a$ will denote the number of vertices adjacent to all of $u, v, w$.

In [6]:
S222 = p.tripleEquations(2, 2, 2, params = {"a": (1, 1, 1)})
S222

0: [0 0 0 0 0 0]
   [0 0 0 0 0 0]
   [0 0 1 0 0 0]
   [0 0 0 0 0 0]
   [0 0 0 0 0 0]
   [0 0 0 0 0 0]

1: [     0      0      0      0      0      0]
   [     0      a      0 -a + 5      0      0]
   [     0      0      0      0      0      0]
   [     0 -a + 5      0 a + 45      0      0]
   [     0      0      0      0      0      0]
   [     0      0      0      0      0      0]

2: [          0           0           1           0           0           0]
   [          0           0           0           0           0           0]
   [          1           0   20*a + 92           0 -20*a + 150           0]
   [          0           0           0           0           0           0]
   [          0           0 -20*a + 150           0  20*a + 200           0]
   [          0           0           0           0           0           0]

3: [          0           0           0           0           0           0]
   [          0      -a + 5           0      a + 45           0           

The number of vertices at distance $5$ from all of $u, v, w$ is a nonnegative integer, implying that the parameter $a$ is either $0$ or $1$.

In [7]:
print(S222[1, 1, 1])
print(S222[5, 5, 5])

a
-12*a + 20


Let us now consider the set $A$ of common neighbours of $u$ and $v$, and the set $B$ of vertices at distance $2$ from both $u$ and $v$. Their sizes are given by the intersection numbers $p^2_{11}$ and $p^2_{22}$.

In [8]:
print(p.p[2, 1, 1])
print(p.p[2, 2, 2])

5
243


By the above, each vertex in $B$ has at most one neighbour in $A$, so there are at most $243$ edges between $A$ and $B$. However, each vertex in $A$ is adjacent to both $u$ and $v$, and the other $k - 2 = 53$ neighbours are in $B$, amounting to a total of $5 \cdot 53 = 265$ edges. We have arrived to a contradiction, and we must conclude that a graph with intersection array $\{55, 54, 50, 35, 10; 1, 5, 20, 45, 55\}$ **does not exist**.